src/components/AdminOrderPage/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Button, Form, Input, Modal } from 'antd';
+import { history } from '../../store';
+import AdminOrderList from '../AdminOrderList';
+
+import './style.less';
+
+interface AdminOrderPageProps {
+  orders: {
+    uuid: string;
+    orderedAt: Date;
+    items: {
+      uuid: string;
+      itemName: string;
+      quantity: number;
+      fulfilled: boolean;
+      price: number;
+      description: string;
+      notes: string;
+    }[];
+  }[];
+  setNote: Function;
+  setFulfill: Function;
+  noteVisible: boolean;
+  setNoteVisible: Function;
+  scratchNote: string;
+  setScratchNote: Function;
+}
+
+const { Search } = Input;
+
+const AdminOrderPage: React.FC<AdminOrderPageProps> = (props) => {
+  const {
+    orders,
+    setNote,
+    setFulfill,
+    noteVisible,
+    setNoteVisible,
+    scratchNote,
+    setScratchNote,
+  } = props;
+
+  return (
+    <div className="admin-orders">
+      <Modal
+        title="Notes"
+        visible={noteVisible}
+        okText="Submit"
+        onOk={() => {
+          setNote(scratchNote);
+          setNoteVisible(false);
+          setScratchNote('');
+        }}
+        onCancel={() => setNoteVisible(false)}
+      >
+        <form>
+          <Form.Item className="note">
+            <Input.TextArea
+              name="note"
+              value={scratchNote}
+              onChange={(event) => setScratchNote(event.target.value)}
+              rows={4}
+            />
+          </Form.Item>
+        </form>
+      </Modal>
+      <div className="store-header">
+        <h2 className="title">Orders</h2>
+        <Button
+          className="admin-redirect-button"
+          onClick={() => {
+            history.push('/store');
+          }}
+        >
+          <b>Store</b>
+        </Button>
+      </div>
```

Comment:
Swap out with header pr after #296 is merged in

Reply:
I assume this means that either we block this PR until the store header is in, or we merge it and make an issue to add the store header later on. For now, I'll just resolve this, since there's nothing to do here yet.